google/cloud/stream_range.h

```diff
@@ -85,13 +91,13 @@ class StreamRange {
   template <typename U>
   static constexpr bool IsMoveNoexcept() {
     return noexcept(StatusOr<U>(std::declval<U>()))&& noexcept(
-        StreamReader<U>(std::declval<StreamReader<U>>()));
+        internal::StreamReader<U>(std::declval<internal::StreamReader<U>>()));
   }
 
  public:
   /// An input iterator for a `StreamRange<T>`
```

Comment:
nit: maybe remind users that this is not part of the public API and they should be using the `iterator` alias below.

Reply:
Good idea. Done.